src/main/java/org/myrobotlab/service/config/ServoConfig.java

```diff
@@ -0,0 +1,22 @@
+package org.myrobotlab.service.config;
+
+public class ServoConfig extends Config {
+
+  public Boolean autoDisable;
+  public String controller;
+  public Double currentOutputPos;
+  public Boolean enabled;
+  public Boolean idleDisabled;  
+  public Integer idleTimeout;
+  public Boolean isBlocking;
```

Comment:
isn't this a detail of a movement?  we shouldn't have  isBlocking in the config.

Reply:
agree